query-service-impl/src/test/resources/application.conf

```diff
@@ -2,6 +2,12 @@ service.name = "query-service"
 service.port = 8090
 service.admin.port = 8091
 service.config = {
+  attribute.client = {
```

Comment:
Where are you using these? I don't see the code to validate attributes yet.

Reply:
These actually aren't used yet, they're used in the following change for projections - that change drove this one, so I split this part off to shrink the review. Should have probably removed these.